packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -436,6 +436,10 @@ class FlutterValidator extends DoctorValidator {
         frameworkVersion,
         _flutterRoot(),
       )));
+      messages.add(ValidationMessage(_userMessages.flutterUpstreamRepositoryUrl(version.repositoryUrl ?? 'unknown')));
+      if (_platform.environment.containsKey('FLUTTER_GIT_URL')) {
```

Comment:
is this necessary given the previous line?

Reply:
changed my mind, I think this is good to have as a reminder in case users forget they set it. LGTM